dashboard/config/locales/en.yml

```diff
@@ -456,6 +456,9 @@ en:
     upgrade:
       unsupported_message: "Your browser is not supported. Please upgrade your browser to <a href='%{supported_browsers_url}', target='_blank'>one of our supported browsers</a>. You can try viewing the page, but expect functionality to be broken."
       link: "Learn more"
+      applab_unsupported_tablet: "App Lab works best on a desktop or laptop computer with a mouse and keyboard.  You may experience issues using this tool on your current device."
```

Comment:
Just to stay consistent with the rest of the content on the site, can we remove the double space? :D